scripts/docker/run.sh

```diff
@@ -0,0 +1,19 @@
+
+# TODO: need a better one
+healthcheck(){
+    while true; do (echo -e 'HTTP/1.1 200 OK\r\n\r\n 1') | nc -lp 1111 > /dev/null; done
+}
+
+find /www/hubs/ -type f -name *.html -exec sed -i "s/{{rawhubs-base-assets-path}}\//https:\/\/${SUB_DOMAIN}-assets.${DOMAIN}\/hubs\//g" {} \;           
+find /www/hubs/ -type f -name *.html -exec sed -i "s/{{rawhubs-base-assets-path}}/https:\/\/${SUB_DOMAIN}-assets.${DOMAIN}\/hubs\//g" {} \; 
+find /www/hubs/ -type f -name *.css -exec sed -i "s/{{rawhubs-base-assets-path}}\//https:\/\/${SUB_DOMAIN}-assets.${DOMAIN}\/hubs\//g" {} \; 
+find /www/hubs/ -type f -name *.css -exec sed -i "s/{{rawhubs-base-assets-path}}/https:\/\/${SUB_DOMAIN}-assets.${DOMAIN}\/hubs\//g" {} \;             
+anchor="<!-- DO NOT REMOVE\/EDIT THIS COMMENT - META_TAGS -->" 
+for f in /www/hubs/pages/*.html; do 
+    for var in $(printenv); do 
+    var=$(echo $var | cut -d"=" -f1 ); prefix="turkeyCfg_"; 
+    [[ $var == $prefix* ]] && sed -i "s/$anchor/ <meta name=\"env:${var#$prefix}\" content=\"${!var//\//\\\/}\"\/> $anchor/" $f; 
```

Comment:
```sh
${!var//\//\\\/}
```
Is this substitution actually doing what you intended? I stared at it for a while and tried to run it myself in a terminal, but I can't quite understand what the intent was. In fact, I think it might be wrong at the moment. Might be helpful to explain it in a comment in this script as well, since it's hard to read without thinking about it for a while.

Reply:
yeah substitution part is just swapping `/` for `\/`, for the slashes in `base_assets_path`
both sed and bash parameter substitution do not like `/` so it needs to be escaped into `\/`
sed does not like `\` either, so the `\/` part end up got escaped into `\\\/`